### ISSUE TITLE
About VisualAtom learning parameters.

### ISSUE THREAD

**Issue by tomoki87** [masora1030/CVPR2023-FDSL-on-VisualAtom#2]
Thank you for your great research.
I am interested in the experiment and would like to reproduce the results.
I understand that the "Pre-training" described in the README is for VisualAtom-21k.
If you have changed the training parameters in the case of Visualatom_1k (case like vit_tiny_with_visualatom_1k.pth.tar in "Pre-trained models"), could you tell me the parameters? ?
Thank you.
 
素晴らしい研究をありがとうございます。
実験に興味があり、結果の再現をしたいと考えています。
READMEの "Pre-training "記載されているのはVisualAtom-21kの場合だと理解しています。
Visualatom_1kの場合("Pre-trained models”でvit_tiny_with_visualatom_1k.pth.tarのようなケース)で学習パラメタを変更されていましたら、パラメタを教えていただけますでしょうか？
よろしくお願いいたします。

**Comment by masora1030** [comment]
@tomoki87 

Hi, tomoki87,
Thank you very much for your interest in our research.
I apologize for the delay in replying to you.
For each hyperparameters for each category size of the dataset, please refer to Table 3 in [Supplementary Material](https://masora1030.github.io/Visual-Atoms-Pre-training-Vision-Transformers-with-Sinusoidal-Waves/CVPR2023_VisualAtom_FDSL_Supplementary_Material.pdf).
If you have any other questions, please feel free to ask.
Thank you in advance.

我々の研究に関心を持っていただき誠にありがとうございます。
返信が遅くなって申し訳ございません。
データセットのカテゴリサイズごとの各ハイパーパラメータについては、[Supplementary Material](https://masora1030.github.io/Visual-Atoms-Pre-training-Vision-Transformers-with-Sinusoidal-Waves/CVPR2023_VisualAtom_FDSL_Supplementary_Material.pdf)のTable 3に記載があるので、そちらをご参照いただますと幸いです。
他にも疑問点等ありましたら、お気軽にお尋ねください。
どうぞよろしくお願いいたします。

**Comment by tomoki87** [comment]
@masora1030 

Thank you for your response. 
From the description of each hyperparameter you provided, I understand the detailed settings. 
This will be an additional question, but could you also tell me the number of batches per gpu when learning with VisualAtom-1k and 21k?
I would like to confirm that the 64 settings in the README are correct, since the batch normalization are calculated per gpu. 
Thank you.

ご回答ありがとうございます。
教えていただいた各ハイパーパラメータの記載箇所から、詳細な設定を理解できました。
追加の質問となるのですが、VisualAtom-1kと21kで学習時の1gpuあたりのバッチ数も教えていただけますでしょうか？
バッチ正規化の計算が1gpuごとに行われる関係で、こちらの設定がREADME記載の64のままで正しいか確認したいと考えております。
よろしくお願いいたします。

**Comment by masora1030** [comment]
@tomoki87 

I am very sorry for the delay in replying.
The batch size per GPU (local batch size) is 64 for both 1k and 21k.
However, it should be noted that the global batch size is different due to the fact that the 1k uses 16 GPUs and the 21k uses 128 GPUs.
I will comment on other issues from you as well. Please wait a little.

返信遅れまして大変申し訳ございません。
1kでも21kでも1GPUあたりのバッチサイズ（ローカルバッチサイズ）は64となっております。
ただし、1kでは16GPU、21kでは128GPU使っている関係でグローバルバッチサイズが異なることに注意する必要があります。
他のissueについてもコメントしていきます。少々お待ちください。

**Comment by tomoki87** [comment]
@masora1030 

Thank you for your response. 
I understand about the local batch size.
I have asked many questions on other issues and would appreciate your answers.
Thank you very much.

ご回答いただきありがとうございます。
ローカルバッチサイズについて理解できました。
他のissueでもたくさん質問させていただいておりますが、回答いただけますと幸いです。
よろしくお願いいたします。
